py3status/modules/stopwatch.py

```diff
@@ -55,6 +55,8 @@ def post_config_hook(self):
         self.time_state = None
         self.color = None
         self.paused = False
+        self.button_reset = 3
+        self.button_toggle = 1
```

Comment:
Move them up to line 50 `# available configuration parameters`

Reply:
done